Shared/LocationController.cpp

```diff
@@ -417,45 +394,12 @@ void LocationController::setIconDataPath(const QString& dataPath)
 /*!
   \brief Returns the URL of the model symbol used for location display.
  */
-QUrl LocationController::modelSymbolPath() const
+QImage LocationController::iconSymbol() const
```

Comment:
```suggestion
QImage LocationController::iconImage() const
```
Also need to update the doc for the method